docs/changelogs/v4.0.0.md

```diff
@@ -0,0 +1,559 @@
+
+#2338 Vis: Fix bug with vtk writing of non-square wave surface (@andrew-platt)
+
+#2350 Vis: Fix another bug with vtk writing of non-square wave surface (@luwang00)
+
+#2431 Enable linearization with MHK turbines (@andrew-platt)
+
+#2445 ExtInfw: relocate initialization to after `AD_Init` (@andrew-platt)
+
+#2478 [BugFix] Incorrect value of AirDens passed to `ExtInfw_Init` from `FAST_Subs.f90` (@andrew-platt)
+
+
+### OpenFAST interfaces
+
+#### OpenFASTcpp
+* Interface library changes
+
+#2245 Fix restart file index and nc write (@ndevelder)
+
+
+#### Simulink
+
+#1703 Add tests for `FAST_SFunc` using Matlab action to integrate with CI (@deslaughter)
+
+
+
+## Modules
+
+### Multiple
+There are several pull requests that affected multiple modules.  Important features include the intial development of aeromaps (for controls development), initial development of the reduced order modules of _AeroDisk_ and _Simplefied-ElastoDyn_ (overviews of each below).
+
+
+#1295 Reduced order models: AeroDisk and Simplified-ElastoDyn (SED) (@andrew-platt)
+
+#1629 MHK: add parameters for readability (@bjonkman)
+
+#1631 Initial AeroMap changes for ElastoDyn and BeamDyn (@bjonkman)
+
+#2203 Support for large platform yaw offset in OpenFAST (ED, HD, SD) (@luwang00)
+
+#2254 Minor changes to FF input descriptions, re-enable test cases, update Simulink string compare (@andrew-platt)
+
+#2332 Minor bugfix: ED blade file parsing (no PichAxis column), AD warnings (@andrew-platt)
+
+#2415 Lidar bug fix + other minor changes (@bjonkman)
+
+#2416 SED+ADsk: update registry comments slightly (@andrew-platt)
+
+
+
+### AeroDyn 14 (deprecated, option replaced by AeroDisk)
+_AeroDyn14_ has been removed from the code base as it has been superceded by _AeroDyn15_ (now called simply _AeroDyn_) for many years.
+
+#2267 AeroDyn14 removal (@andrew-platt)
+
+
+### AeroDisk
+_AeroDisk_ is a new module for a disk actuator aerodynamic module.  This module assumes a rigid disk with only a few degrees of freedom for modeling turbine aerodynamics with a simple actuator disk.  This is useful for modeling turbines that are not of interest in very large wind farms modeled by _FAST.Farm_, but whose wake dynamics are needed for turbines further into the wind farm that are of interest.  We do not recommend ising this module with standalone _OpenFAST_ simulations.
+* New input file
+
+#2575 ADsk: correction to disk average velocity equations (@andrew-platt)
+
+
+
+### AeroDyn
+_AeroDyn_ improvements include a new input file, nacelle drag, visualization improvements, unsteady aero for tailfins, multi-rotor support in the driver and interface library, a simple ground effect model for _OLAF_ wakes, using _InflowWind_ pointers for data acces, and many bug fixes.  With the removal of _AeroDyn14_, _AeroDyn15_ is now simply referred to as _AeroDyn_ in the input files and code (documentation may not be fully updated).
```

Comment:
Add: "new options in BEM to improve aerodynamics for skewed and sheared inflow and coned rotors".
Add: "an improved unsteady airfoil aerodynamics driver".
Actually, multirotor capability has been supported by the standalone AeroDyn driver for a while; is the change mentioned only for the interface library?  Suggest clarifying.

Reply:
yes, the multirotor option is new to the library interface.  I'll clarify that.